packages/core/admin/admin/src/content-manager/utils/getDisplayName.js

```diff
@@ -6,9 +6,15 @@
  * @property {string} username
  * @property {string} email
  *
- * @type {(user: AdminUserNamesAttributes, formatMessage: import('react-intl').formatMessage) => string}
+ * @type {(user: AdminUserNamesAttributes | null, formatMessage: import('react-intl').formatMessage) => string}
  */
-const getDisplayName = ({ firstname, lastname, username, email }, formatMessage) => {
+const getDisplayName = (user, formatMessage) => {
+  if (!user) {
+    return '-';
+  }
```

Comment:
I'd prefer if we catch that before calling the function, because it is used in many contexts, where you don't want a `-` fallback. Wdyt?

Reply:
Cool. If you want you could throw an error in case `user` wasn't passed - or wait until it is converted to TS :)